### DISCUSSION TITLE
組立後にトラックボールが認識しない KeyBall39

### DISCUSSION THREAD

**Discussion by apro0666** [Yowkees/keyball#359] [Q&A]
初めまして。
遊舎工房よりKeyBall39を購入し昨日組み立てました。
右手で作業すると肩が痺れてくるためトラックボールを左手側に設置しております。

左右のProMicroにKeyball39_viaをFrashし、
OLEDディスプレイ上で全Keyが認識されることを確認しましたが、
トラックボールが全く認識せずポインターが反応しません。

原因特定のための切り分け方法をお教え願えないでしょうか。
また、当現象によくある組み立てミスの確認方法、作業手順の間違い
などあればご教示ください。

**Comment by apro0666** [comment]
また、remapにて「＋KEYBORD」を押下すると下記の様にエラーメッセージが表示されます。
The VIA protocol version of the connected keyboard is 9. Use "Remap for QMK 0.18" to customize the keyboard.

![image](https://github.com/Yowkees/keyball/assets/10240599/a4c2f68d-e73d-4a36-91b4-146f1ab629da)

**Reply by apro0666** [reply]
Keyball39_via_v1.2.0のインストールにより下記メッセージは表示されなくなりました。
「The VIA protocol version of the connected keyboard is 9. Use "Remap for QMK 0.18" to customize the keyboard.」

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

トラックボール不具合の原因で可能性が高いのは、
7Pコンスルー近くのジャンパミスか、ProMicro裏の半田付けミスです。

下記画像の同色どうしが導通しているかご確認ください。
また、ProMicro裏で隣の端子とショートしているものがないかご確認ください。ここは隣と距離が近いのでショートしやすいです。

改善しない場合、トラックボール基板の表裏やＩＣの向き違い等の可能性もあり、半田付けした基板のお写真を見せていただければ幸いです。
以上、お手数ですがご確認お願いいたします。
![kb39lefttraball](https://github.com/Yowkees/keyball/assets/84209772/6977032c-40e7-4243-af97-866135051b2d)

**Reply by apro0666** [reply]
対処方法のご連絡ありがとうございます。
見た目綺麗に半田付けされていたのですが、ご指摘の8箇所の半田付けをやり直しトラックボールの動作を確認しました。

**Comment by Yowkees** [comment]
無事に動作したようで安心しました。
ぜひ末永くご愛用いただければ幸いです。
